### DISCUSSION TITLE
Support for the options real-time data?

### DISCUSSION THREAD

**Discussion by shaferica** [Petersoj/alpaca-java#152] [General]
I'm trying to connect to the Alpaca options real-time feed directly (wss://stream.data.alpaca.markets/v1beta1/opra) but have been having issues with getting data back once authenticated (some nuance with MsgPack). So I looked at this Alpaca Java API, but I can't find any way to connect to the options stream here, subscribe to channels, etc. Does this SDK support Options streaming data?

**Comment by Petersoj** [comment]
While options trading and historical data are supported in this library, options data streaming is not yet supported. I haven't found the time to implement it yet. Feel free to open a PR for it though!

**Reply by shaferica** [reply]
OK TY, I'll get back to you once I get an implementation working. So far I'm on life support from Alpaca's support team. It doesn't look like they've given this feature much love yet.
